make/include/contrib.defs

```diff
@@ -94,7 +94,7 @@ define import.CONTRIB.defs
 
     $(1).CONFIGURE.args.dir    = cd $$(1);
     $(1).CONFIGURE.args.host   = --host=$$(1)
-    $(1).CONFIGURE.args.build  = --build=$$(1)
+    $(1).CONFIGURE.args.build  = --build=x86_64-unknown-linux-gnu
```

Comment:
Do you know what contribs have this problem?  I think the right way would be to modify individual module.defs for the contribs that have problems.  In the module.defs file, you should be able to do something like
```
ifeq (msys,$(BUILD.system))
    FFMPEG.CONFIGURE.args.build = --build=x86_64-unknown-linux-gnu
endif